AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -306,13 +306,13 @@ private async Task SendJudgeEmailIfNeeded(HearingDetailsResponse updatedHearing,
         {
             if (newParticipantList.Any())
             {
-                _logger.LogDebug("Sending email notification to the participants");
+                _logger.LogInformation("Sending email notification to the participants");
```

Comment:
Log levels in PROD may not log LogInformation. Please check the log levels in PROD first.

Reply:
AppInsights will capture info and higher :)